### ISSUE TITLE
Release 1.16.0

### ISSUE THREAD

**Issue by enricoros** [enricoros/big-AGI#530]
## Release checklist:

- [x] Create a new [Release Issue](https://github.com/enricoros/big-AGI/issues/new?assignees=enricoros&projects=enricoros/4&template=maintainers-release.md&title=Release+1.16.0)
  - [x] Replace 1.15.1 with the _former_ release, and _1.16.0_ with THIS
- [x] Update the [Roadmap](https://github.com/users/enricoros/projects/4/views/2) calling out shipped features
- [x] Create and update a [Milestone](https://github.com/enricoros/big-agi/milestones) for the release
  - [x] Assign this task
  - [x] Assign all the shipped roadmap Issues
  - [x] Assign the relevant [recently closed Isssues](https://github.com/enricoros/big-agi/issues?q=is%3Aclosed+sort%3Aupdated-desc)
- Code changes:
  - [x] Create a release branch 'release-x.y.z': `git checkout -b release-1.16.0`
  - [x] Create a temporary tag `git tag v1.16.0 && git push opensource --tags`
  - [x] Create a [New Draft GitHub Release](https://github.com/enricoros/big-agi/releases/new), and generate the automated changelog (for new contributors)
  - [x] Update the release version in package.json, and `npm i`
  - [x] Update the in-app News version number
  - [x] Update in-app News [src/apps/news/news.data.tsx](/src/apps/news/news.data.tsx)
  - [x] Update in-app Cover graphics
  - [x] Update the README.md with the new release
  - [x] Copy the highlights to the [docs/changelog.md](/docs/changelog.md)
- Release:
  - [x] merge onto main `git checkout main && git merge --no-ff release-1.16.0`
  - [x] re-tag `git tag -f v1.16.0 && git push opensource --tags -f`
  - [x] verify deployment on Vercel
  - [x] verify container on GitHub Packages
  - [x] update the GitHub release
  - [x] push as stable `git push opensource main:main-stable`
- Announce:
  - [x] Discord announcement
  - [ ] Twitter announcement

### Links

- Milestone: https://github.com/enricoros/big-AGI/milestone/16
- GitHub release: https://github.com/enricoros/big-AGI/releases/tag/v1.16.0
- Former release task: #...

## Artifacts Generation

```markdown
You help me generate the following collateral for the new release of my opensource application called big-AGI. The new release is 1.16.0.
To familiarize yourself with the application, the following are the Website and the GitHub README.md.
```

- paste the URL: https://big-agi.com
- drag & drop: [README.md](https://raw.githubusercontent.com/enricoros/big-AGI/main/README.md)

```markdown
I am announcing a new version, 1.16.0.
For reference, the following was the collateral for 1.15.1 (Discord announcement, GitHub Release, in-app-news file news.data.tsx).
```

- paste the former: `discord announcement`,
- `GitHub release`,
- `news.data.tsx`,
- `changelog.md`

```markdown
The following are the new developments for 1.16.0:

- ...
- git log --pretty=format:"%h %an %B" v1.15.1..v1.16.0 | clip
```

- paste the link to the milestone (closed) and each individual issue (content will be downloaded)
- paste the output of the git log command

### news.data.tsx

```markdown
I need the following from you:

1. a table summarizing all the new features in 1.16.0 with the following columns: 4 words description (exactly what it is), short description, usefulness (what it does for the user), significance, link to the issue number (not the commit)), which will be used for the artifacts later
2. then double-check the git log to see if there are any features of significance that are not in the table
3. then score each feature in terms of importance for users (1-10), relative impact of the feature (1-10, where 10 applies to the broadest user base), and novelty and uniqueness (1-10, where 10 is truly unique and novel from what exists already)
4. then improve the table, in decreasing order of importance for features, fixing any detail that's missing, in particular check if there are commits of significance from a user or developer point of view, which are not contained in the table
5. then I want you then to update the news.data.tsx for the new release
```

### release name

```markdown
please brainstorm 10 different names for this release. see the former names here: https://big-agi.com/blog
```

You can follow with 'What do you think of Modelmorphic?' or other selected name

### cover images

```markdown
Great, now I need to generate images for this. Before I used the following prompts (2 releases before).

// An image of a capybara sculpted entirely from black cotton candy, set against a minimalist backdrop with splashes of bright, contrasting sparkles. The capybara is using a computer with split screen made of origami, split keyboard and is wearing origami sunglasses with very different split reflections. Split halves are very contrasting. Close up photography, bokeh, white background.
import coverV113 from '../../../public/images/covers/release-cover-v1.13.0.png';
// An image of a capybara sculpted entirely from black cotton candy, set against a minimalist backdrop with splashes of bright, contrasting sparkles. The capybara is calling on a 3D origami old-school pink telephone and the camera is zooming on the telephone. Close up photography, bokeh, white background.
import coverV112 from '../../../public/images/covers/release-cover-v1.12.0.png';

What can I do now as far as images? Give me 4 prompt ideas with the same style as looks as the former, but different scene or action
```

### Readme (and Changelog)

```markdown
I need you to update the README.md and the with the new release.
Attaching the in-app news, with my language for you to improve on, but keep the tone.
```

### GitHub release

```markdown
Please create the 1.16.0 Release Notes for GitHub, following the format of the 1.15.1 GitHub release notes attached before.
Use a truthful and honest tone, understanding that people's time and attention span is short.
Today is 2024-XXXX-YYYY.
```

Now paste-attachment the former release notes (or 1.5.0 which was accurate and great), including the new contributors and
some stats (# of commits, etc.), and roll it for the new release.

### Discord announcement

```markdown
Can you generate my 1.16.0 big-AGI discord announcement from the GitHub Release announcement?
Please keep the formatting and stye of the discord announcement for 1.15.1, but with the new messaging above.
```

**Comment by enricoros** [comment]
![DALL·E 2024-05-08 14 01 41 - A landscape JPG image of a capybara made entirely of clear, translucent crystal, wearing oversized black sunglasses, sitting at a sleek](https://github.com/enricoros/big-AGI/assets/32999/02472393-96d5-47b8-96f4-15d16b193c14)

**Comment by enricoros** [comment]
Thanks @myobfee, it's the only way to do it right, and as far as the LLM prompts, I like to go through the pains of doing something manually, but you can see where that may be going in big-AGI ;) My target for a release is to go down from the 2-3hrs of time to a 5 min of manual time.

**Comment by enricoros** [comment]
GitHub release
---
# 🚀 big-AGI 1.16.0 - Crystal Clear

### Clarity and transparency in your AI interactions.

**big-AGI 1.16.0** - **Crystal Clear** - brings enhancements and new features designed to refine your experience and expand functionality, ensuring that every interaction with big-AGI is smoother, more intuitive, and simply better - as we always do. We hope you'll love the updated models across the board, and the new cost tracking (please refresh the models to load the latest prices), now active on Anthropic, OpenAI, and OpenRouter, which will translate to saving money, as anything that can be measured can be optimized for.

![release-cover-v1.16.0](https://github.com/enricoros/big-AGI/assets/32999/02472393-96d5-47b8-96f4-15d16b193c14)

## What's Changed

- **Chat Cost Estimation**: the token counter will display a cost estimation for supported networks. hover for token info. disable in Labs
- Enhanced [Beam](https://big-agi.com/blog/beam-multi-model-ai-reasoning) based on your feedback: auto-selection and remembering networks and UX simplifications #470 
- Major improvements to the **Auto-Diagrams Tool**
- **Save/Load Chat Files** with Ctrl+S and Ctrl+O #466 
- @joriskalz: **YouTube Transcriber** persona: transcribe and chat with YouTube videos (un-hide it from the selector) #500
- @mludvig: **3x faster on Apple silicon** with Docker multi-arch builds #507
- More: Perfect formulas/LaTeX rendering #508, soft-wrap code #517, chat text selection toolbar, dark mode fixes #520
- Doc corrections by @dandv and @PrivTEC and in #521 #522 #525 
- Model Updates: Latest improvements from Anthropic, Groq, Ollama, OpenAI, OpenRouter, and Perplexity.
- For Developers: updated LLM data structures
- [Full Changelog](https://github.com/enricoros/big-AGI/compare/v1.15.1...v1.16.0)

## New Contributors
* A warm welcome and thank you to @mludvig, @PrivTEC, and @dandv for their first contributions to big-AGI!

## In Summary
Transparency and user experience improvements, for an improved quality of life. We are thankful to be able to merge contributions for the community, including a new YouTube Transcriber persona and the faster multi-arch (ARM, x86) builds.

Upgrade today and experience the clearer, faster, and more intuitive big-AGI. Available via code, Docker, or on the free instance on [big-agi.com](https://big-agi.com). 🚀

**Comment by enricoros** [comment]
Discord
---
@everyone :sparkles: **big-AGI 1.16.0 - Crystal Clear** Deployed

:partyblob~1: **Clarity and transparency in your AI interactions.** :partyblob~1:

You have 1.6.0 now. This release focuses on transparency and user experience improvements, a better and more useful Beam based on your feedback (incl. @sisuaave , @k0880k_39229 and ALL!), and the headlining feature, which is **Chat Cost Estimation**.

- :moneybag: **Chat Cost Estimation** for supported networks (Anthropic, OpenAI, OpenRouter - please refresh your models to see the costs :warning:). Hover the price for a detailed breakdown of tokens and costs
- :ramen:  **Beam v1.1 Improvements** incl. Auto-Model-selection, last models memory, and UX simplifications (from users feedback / UX research)
- :youtube: **YouTube Transcriber Persona**: Transcribe and chat with YouTube videos (unhide in Personas selector)
- :FastChat:  **3x Faster on Apple Silicon**: Thanks to @mludvig's Docker multi-arch builds
- Major improvements to the **Auto-Diagrams Tool**
- Save/Load Chat Files: Use Ctrl+S and Ctrl+O on Desktop
- Updated Anthropic, Groq, Ollama, OpenAI, OpenRouter, and Perplexity models
- Improved formulas/LaTeX rendering, soft-wrap code, chat text selection toolbar, dark mode fixes, and much more

**For the Devs:**
- Updated the LLMs data structure to enable price/perf ranking and more
- Welcome and thanks to @mludvig, @PrivTEC, and @dandv for their first contributions!

:rocket: Experience **Crystal Clear** - available now on GitHub and served to you on https://big-agi.com. :rocket:
